tensorflow/core/common_runtime/gpu/gpu_process_state.h

```diff
@@ -89,6 +89,10 @@ class GPUProcessState {
     return gpu_allocators_.size();
   }
 
+  Allocator* GetConstructedGPUAllocator(int ordinal) {
```

Comment:
Why not just use the existing `GPUProcessState::GetGPUAllocator`? This seems like a hack.

Reply:
Oh, I thought `GPUProcessState::GetGPUAllocator` was only used to create a new allocator. And it is fixed now.